jolpica_api/ergastapi/serializers.py

```diff
@@ -118,6 +107,45 @@ def get_qualifying(self, race: Round):
     def get_sprint(self, race: Round):
         return self.get_session_date_time(race, SessionType.SPRINT_RACE)
 
+    def get_sprint_qualifying(self, race: Round):
+        return self.get_session_date_time(race, SessionType.SPRINT_QUALIFYING1)
+
+
+    class Meta:
+        model = Round
+        fields = ["season", "round", "url", "raceName", "Circuit", "date", "time"]
+
+
+class ListRaceSerializer(serializers.ListSerializer):
+    def to_representation(self, data: list[Round]) -> dict:
+        races = list()
+        for _round in data:
```

Comment:
nit-pick: no need for `_round` over `round` as this isn't a class attribute or accessible outside the function

Reply:
I usually work by the rule that you just never redefine internal names if somehow possible. But I can change that here, if you prefer.

Reply:
Thats a good point I hadn't considered.
Lets use a trailing underscore instead to align [with pep8](https://peps.python.org/pep-0008/#descriptive-naming-styles)

Reply:
good point